NorthAIBase.cpp

```diff
@@ -0,0 +1,51 @@
+#include "NorthAIBase.h"
+#include "AIHelper.h"
+#include "AIPlayer.h"
+#include "HFL/Source/HFL.h"
+#include "OP2Helper/OP2Helper.h"
+#include "Outpost2DLL/Outpost2DLL.h"
```

Comment:
Since Outpost2DLL is included in the header file, you can remove it from the .cpp file include list.